doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -103,7 +104,200 @@ It covers the test cases against compute management operations.
 It covers the test cases against orchestration operations.
 
 <a name="4.3.5"></a>
-### 4.3.5 opensource VNF onboarding and testing
+### 4.3.5 OpenStack API benchmarking
+
+[Rally](https://opendev.org/openstack/rally) is tool and framework that allows
+to perform OpenStack API benchmarking.
+
+Here are the Rally-based test cases proposed by
+[Functest Benchmarking CNTT](https://git.opnfv.org/functest/tree/docker/benchmarking-cntt/testcases.yaml)
+- [rally_full](http://artifacts.opnfv.org/functest/IR6NYE2BYC8W/functest-opnfv-functest-benchmarking-hunter-rally_full-run-324/rally_full/rally_full.html):
+  Functest scenarios iterating 10 times the mainline Rally scenarios
+- [rally_jobs](http://artifacts.opnfv.org/functest/IR6NYE2BYC8W/functest-opnfv-functest-benchmarking-hunter-rally_jobs-run-328/rally_jobs/rally_jobs.html):
+  Neutron scenarios executed in the OpenStack gates
+
+At the time of writing, no KPI is defined in
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
+which would have asked for an update of the default SLA proposed in
+[Functest Benchmarking CNTT](https://git.opnfv.org/functest/tree/docker/benchmarking-cntt/testcases.yaml)
```

Comment:
We should atleast have `failure_rate` as KPI. Without any KPI, it is not a performance test.

Reply:
@deebhatia: It fails if one iteration fails.
At the time writing, we can't decrease/increase the default timeouts as no kpi has been defined yet in CNTT. The default values have been used for years in OPNFV.
Be free to dive into rally before comments